pages/learn/develop/dockerfile.md

```diff
@@ -23,9 +23,9 @@ Typically you will only need to use 4 instructions - [FROM][from], [RUN][run] an
 
 * [ADD][add] copies files from the current directory into the container, e.g. `ADD <src> <dest>`. Note that if `<dest>` doesn't exist, it will be created for you, e.g. if you specify a folder. It also allows the <src> to be a url, and if the <src> is a recognised compression format, it will unpack it for you.
 
-* [COPY][copy] is very similar to [ADD][add], but without the compression and url functionality. According to [docker best practise][docker-best-practise] you should always use [COPY][copy] unless the auto-extraction capability of [ADD][add] is needed.
+* [COPY][copy] is very similar to [ADD][add], but without the compression and url functionality. According to [Docker best practices][docker-best-practise], you should always use [COPY][copy] unless the auto-extraction capability of [ADD][add] is needed.
```

Comment:
`Docker best practices` is better as `Dockerfile best practices`, since the original title is `Best practices for writing Dockerfiles`? Or maybe we should use the original title?

Also `[docker-best-practise]` is redefined as plural below. Maybe even better we can use the original slug `[dockerfile_best-practices]`?